### PR TITLE
Adding HTTP status codes cheat sheet

### PR DESCRIPTION
**Pull request by brianbs**
(please add PR template here)

------

IA Page: https://duck.co/ia/view/http_codes_cheat_sheet

### CODE REVIEW

**Comment by MrChrisW** [comment]
Hi @brianbs, thanks for your contribution. Do you mind filling out the [Goodie Pull Request Template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_goodie.md) and adding it to your initial comment. This will provide the community a quick way to review your CheatSheet Goodie. Thanks :+1:

File: share/goodie/cheat_sheets/http-status-codes.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "http_status_codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/http-status-codes.json]: Please update to `http_codes_cheat_sheet`

File: share/goodie/cheat_sheets/http-status-codes.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    "name": "HTTP Status Codes",
+    "description": "A list of HTTP status codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/http-status-codes.json]: We can remove this, it's a little redundant with the title ;)

Comment (moollaza) [inline share/goodie/cheat_sheets/http-status-codes.json]: Or rather improve it to be a little more descriptive: "HTTP request status codes and their meanings"

I'm happy either way.

**Comment by moollaza** [comment]
Ping! @brianbs are you still interested in finishing this! We'd love to see it go live!

**Comment by brianbs** [comment]
@moollaza I am!  WIll get make the changes tonight or tomorrow night!

**Comment by moollaza** [comment]
@brianbs awesome! Thanks a lot :+1:

**Comment by moollaza** [comment]
@brianbs Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by altern8tif** [comment]
@brianbs I'm more than happy to continue this PR if you're caught up with other stuff.

**Comment by moollaza** [comment]
@altern8tif It seems this PR has been abandoned :frowning: 

You're welcome to finish it up if you'd like :smiley:

**Pull request closed without merging**